urwid/widget/padding.py

```diff
@@ -40,12 +40,14 @@ def __init__(
         self,
         w: WrappedWidget,
         align: (
-            Literal["left", "center", "right"] | Align | tuple[Literal["relative", WHSettings.RELATIVE], int]
+            Literal["left", "center", "right"]
+            | Align
+            | tuple[Literal["relative", WHSettings.RELATIVE, "fixed left", "fixed right"], int]
```

Comment:
type annotate supported (not add to docs, just simplify work with existing code)